bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/BookieServer.java

```diff
@@ -124,6 +124,7 @@ public BookieServer(ServerConfiguration conf, StatsLogger statsLogger)
      */
     public void setExceptionHandler(UncaughtExceptionHandler exceptionHandler) {
         this.uncaughtExceptionHandler = exceptionHandler;
+        this.bookie.setExceptionHandler(exceptionHandler);
```

Comment:
nit: since bookie is not final, there can be possibility `bookie` is null if these methods are not called in correct order. can we add `final` modifier to `bookie` field?

Reply:
Fixed